st2actions/st2actions/cmd/actionrunner.py

```diff
@@ -46,9 +46,6 @@ def _setup():
     register_common_signal_handlers()
 
     # 4. Register internal triggers
-    # Note: We need to do import here because of a messed up configuration
```

Comment:
importing in the right order makes the config issue go away?

Crap. This gives me c/c++ import order nightmares all over again!